File: 217-contains-duplicate/contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        int n=nums.size();
       unordered_map<int,int>s;
       for(int i=0;i<n;i++){
        if(s[nums[i]]>=1){
            return true;
        }
        s[nums[i]]++;
       }

    return false;
    }
};

File: 217-contains-duplicate/contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        n=len(nums)
        a=set()
        for num in nums:
            if num in a:
                    return True
            a.add(num)
        return False